nanshe_workflow/data.py

```diff
@@ -118,9 +118,13 @@ def zip_zarr(name):
     with zipfile.ZipFile(name + zip_ext, "w"):
         pass
 
-    with open_zarr(name + zip_ext, "w") as f1:
+    # Opening in append mode to workaround a bug in `flush`.
+    # xref: https://github.com/alimanfoo/zarr/issues/158
+    with open_zarr(name + zip_ext, "a") as f1:
```

Comment:
Opening in append mode to workaround a bug in `flush`. ( https://github.com/alimanfoo/zarr/issues/158 )